README.md

```diff
@@ -48,7 +48,7 @@ Number             | Title                                                    |
 [38](bsip-0038.md) | Add target collateral ratio option to short positions                       | Abit More | Protocol | Installed
 [39](bsip-0039.md) | Automatically approve proposals by the proposer                             | Fabian Schuh | Protocol | Draft
 [40](bsip-0040.md) | Custom active permission                             | Stefan Schießl | Protocol | Accepted - Milestone 1
-[41](bsip-0041.md) | Reduce MSSR of bitCNY from 1.1 to 1.05               | Jerry Liu      | Protocol | Accepted
+[41](bsip-0041.md) | Reduce MSSR of bitCNY from 1.1 to 1.05               | Jerry Liu      | Informational | Superseded by [59](bsip-0059.md)
```

Comment:
I would leave the "Accepted" in (but still add "Superseded")

Reply:
IMHO Superseded implies Accepted or Installed.

Reply:
I feel BSIP41 is very specific to setting a single value and is now obsoleted by the BSIP59 voting process.  It seems somewhat disingenuous to say BSIP41 is accepted, as the value specified is 1.01 currently.  I also like the succinctness of "Superseded by BSIP". In this case, the reader must resort to another source for the current accepted value.

Reply:
Leaving the "Accepted" here would confuse readers, because they may think 1.05 is the final (current) consensus, while it's not.

Reply:
I think that there can be a benefit to distinguishing "superceded" and "accepted" because "Version 2" of a BSIP might supercede "Version 1" but either one may or may not be accepted or installed.

Reply:
So what to do?

Reply:
I think that two or more descriptions can be applied to a BSIP if it is appropriate.